api/config.go

```diff
@@ -105,6 +107,9 @@ func LoadDefaultConfig(args []string) (config Config, err error) {
 	config.FlagPullInterval = 60
 	config.FlagUpdateHealthReportInterval = 60
 
+	// Set default pull timeout to 10 seconds
+	config.FlagPullTimeout = 10
```

Comment:
Do we have any data to support this timeout on large (greater than 50 nodes) clusters?